### DISCUSSION TITLE
Blank Index & Missing Banners

### DISCUSSION THREAD

**Discussion by WickedCPU** [tltneon/lgsl#96] [Q&A] [answered]
I've installed with the latest release (6.1.1) and then removed the install.php. My index.php loads as a blank white page. Tried PHP 7.4 & 8.0 with the same result, are these versions not supported? The admin.php file loads properly.

For the banners, the server's I've added don't have a "Click to show server banners" button anywhere. Does it have to be enabled somewhere?

**Comment by TacTicTow** [comment, marked as answer]
A link to your site would be helpful. I am running php 8.1.1 and works just fine. Do you have any errors on screen or in the logs? 
Not sure if you want to post a redacted version of your config file. That might help as well.

**Reply by WickedCPU** [reply]
I've checked again and it seems to have been a cache or DNS issue causing it to not load correctly. I'm able to see everything now.
